docs/security/OWASP_ZAP_WALKTHROUGH.md

```
@@ -0,0 +1,292 @@
+
+---
+
+## Prerequisites
+
+1. **Applications Running:**
+   - LexFiat: `http://localhost:5173` (dev server)
+   - Arkiver: `http://localhost:5174` (dev server)
+   - Cyrano API: `http://localhost:5001` (if testing API)
+
+2. **ZAP Installed:**
+   - Download from: https://www.zaproxy.org/download/
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)